src/data/calculate.ts

```diff
@@ -124,12 +125,16 @@ export type QueryData = Partial<CalculatorData>
 // Replace any values that no longer exist with empty string (nothing selected).
 // This is used when restoring a previous saved scenario, in case we changed
 // the model in the meantime.
-export const migrateDataToCurrent = (
-  incomingData: Record<string, unknown>,
+// sanitizeData() is used to clean both query parameters in the URL and
+// anything in local storage.
+export const sanitizeData = (
```

Comment:
I'd love to see a test for these functions. I won't block on it, but it would be really good to have a test file with all the past incarnations of stored data formats.

Reply:
(Also, I low-key prefer these functions be in another file, possibly queryParams. I think the reason for them being structured like this is that queryParams.ts is newer than migrateDataToCurrent. Again, I don't insist.)

Reply:
added tests, maybe refactor later?

Reply:
Looks like tests are failing. Are you running the tests locally before pushing? If not, let me know if you need any support getting testing working. Took me a bit to get to to speed on it. If so, then I'm wondering why tests pass locally but not on the server.

Reply:
I did, saw everything pass, and assumed that order of the tests didn't matter (!!!)

Commented out the offending lines, but I'd rather get this in sooner than wait to mock up or clear localStorage to fix it properly.